akka-docs-dev/rst/scala/code/docs/stream/ReactiveStreamsDocSpec.scala

```diff
@@ -135,4 +133,16 @@ class ReactiveStreamsDocSpec extends AkkaSpec {
     assertResult(storage)
   }
 
+  "use a processor" in {
+
+    //#use-processor
+    // An example Processor factory
+    def createProcessor: Processor[Int, Int] = Flow[Int].toProcessor.run()
+
+    val src: Source[Int, Unit] = Source.single(42).viaProcessor(() => createProcessor)
+    val flow: Flow[Int, Int, Unit] = Flow[Int].viaProcessor(() => createProcessor)
```

Comment:
>  -nearly every single method on Flow- -semantically- is via but doesn't carry it in its name.

This is not true. All the non-via methods add exactly one stage, while .via() expresses the intent of adding a prepackaged entity. This is where we disagree, because for me .via() is for adding larger modules (a Flow or Processor), therefore for me it is *not the same*

But let's have a Flow.apply then, that settles this issue.